### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by nmarcetic**
Signed-off-by: nmarcetic <n.marcetic86@gmail.com>

### What does this do?
Updates README file.

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ to the [EdgeX Foundry][edgex] project. It has been made with :heart: by [Mainflu
...
-- Multi-protocol connectivity and protocol bridging (HTTP, MQTT, WebSocket and CoAP)
-- Device management and provisioning
+- Multi-protocol connectivity and bridging (HTTP, MQTT, WebSocket and CoAP)
+- Device management and provisioning (Zero Touch provisioning)
+- Mutual TLS Authentication (mTLS) using X.509 Certificates
 - Fine-grained access control
-- Storage support (Cassandra, InfluxDB and MongoDB)
-- Platform logging and instrumentation support
+- Message persistence (Cassandra, InfluxDB MongoDB and PostgresSQL)
+- Platform logging and instrumentation support (Grafana, Prometheus, Opentracing)
```
Comment (drasko) [inline README.md]: Should be `OpenTracing`

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by blokovi — DISMISSED** [review]

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -21,11 +21,12 @@ to the [EdgeX Foundry][edgex] project. It has been made with :heart: by [Mainflu
...
 ## Features
-- Multi-protocol connectivity and protocol bridging (HTTP, MQTT, WebSocket and CoAP)
-- Device management and provisioning
+- Multi-protocol connectivity and bridging (HTTP, MQTT, WebSocket and CoAP)
+- Device management and provisioning (Zero Touch provisioning)
+- Mutual TLS Authentication (mTLS) using X.509 Certificates
 - Fine-grained access control
-- Storage support (Cassandra, InfluxDB and MongoDB)
-- Platform logging and instrumentation support
+- Message persistence (Cassandra, InfluxDB MongoDB and PostgresSQL)
```
Comment (manuio) [inline README.md]: There is a `,` missing between InfluxDB and MongoDB

File: README.md [hunk 3]
```diff
@@ -21,11 +21,12 @@ to the [EdgeX Foundry][edgex] project. It has been made with :heart: by [Mainflu
...
+- Platform logging and instrumentation support (Grafana, Prometheus, OpenTracing)
```
Comment (manuio) [inline README.md]: Maybe better `Prometheus and OpenTracing)`

**Review by manuio — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -25,8 +25,8 @@ which maintains the project and offers professional services around it.
 - Device management and provisioning (Zero Touch provisioning)
 - Mutual TLS Authentication (mTLS) using X.509 Certificates
 - Fine-grained access control
-- Message persistence (Cassandra, InfluxDB MongoDB and PostgresSQL)
-- Platform logging and instrumentation support (Grafana, Prometheus, OpenTracing)
+- Message persistence (Cassandra, InfluxDB, MongoDB and PostgresSQL)
+- Platform logging and instrumentation support (Grafana, Prometheus and OpenTracing)
```
Comment (drasko) [inline README.md:29]: Maybe add Fluentd and ElasticSearch?

**Review by anovakovic01 — APPROVED** [review]
LGTM!

File: README.md (hunk 4 above)
Comment (drasko) [inline README.md:29]: Maybe add CockroachDB and ScyllaDB?

**Review by manuio — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**